icu4c/source/common/unicode/urename.h

```diff
@@ -586,7 +585,6 @@
 #define ucasemap_getLocale U_ICU_ENTRY_POINT_RENAME(ucasemap_getLocale)
 #define ucasemap_getOptions U_ICU_ENTRY_POINT_RENAME(ucasemap_getOptions)
 #define ucasemap_internalUTF8ToTitle U_ICU_ENTRY_POINT_RENAME(ucasemap_internalUTF8ToTitle)
-#define ucasemap_mapUTF8 U_ICU_ENTRY_POINT_RENAME(ucasemap_mapUTF8)
```

Comment:
Spot-check: This is the name of two C++ functions that have neither `extern "C"` nor are members of a namespace or class. (ucasemap_imp.h, ucasemap.cpp) At least one is used from another "common" file (ucasemap_titlecase_brkiter.cpp).

Since it's not in the "icu" namespace, I think it should get renamed. I wonder why the tool removes it this time.